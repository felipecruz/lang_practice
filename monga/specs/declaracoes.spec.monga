int a;
int b;

void func (int a, int b) {}
int[] get_array (int[] array) {}

int x, y, z;

int[] a1, b2;

char[] a3;
float[] floats;
