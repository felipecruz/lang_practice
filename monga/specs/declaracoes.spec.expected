
 Var Decl Int Name a 
 Var Decl Int Name b 
Func Decl - Return Type: Void 
          - Paramenters Int b, Int a
Func Decl - Return Type: Int[] 
          - Paramenters Int[] array
 Var Decl Int Name z Name y Name x 
 Var Decl Int[] Name b Name a 
 Var Decl Char[] Name a 
 Var Decl Float[] Name floats 
